kaska/tests/test_nn_library.py

```diff
@@ -1,52 +1,58 @@
 #!/usr/bin/env python
 """Test reproject data"""
 import os
-
 import sys
-sys.path.append("../")
-
 import pytest
 import gdal
-import numpy as np
+from ..inverters import get_emulators, get_emulator
+from ..inverters import get_inverters, get_inverter
 
+sys.path.append("../")
```

Comment:
Check whether the tests run without this line. I believe they should.

Reply:
Yep they do